### PR TITLE
Adjoh/merge 3.1.3 to master

### PR DESCRIPTION
**Pull request by AdamBJohnsonx**
Please make sure every Pull Request has the following information:

* A reference to a related issue in your repository (mandatory)
* A description of the changes proposed in the pull request
* @mentions of the person required for reviewing proposed changes (optional)

### CODE REVIEW

**Review by shahzaibj — APPROVED** [review]

File: adal/build.gradle [hunk 1]
```diff
@@ -133,7 +134,7 @@ dependencies {
     snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '3.0.6', changing: true)
 
     // 'dist' flavor dependencies
-    distApi("com.microsoft.identity:common:3.0.6") {
+    distApi("com.microsoft.identity:common:3.4.5-RC2") {
```
Comment (shoatman) [inline adal/build.gradle:137]: Are we using RC2 in the adal release?

Comment (AdamBJohnsonx) [inline adal/build.gradle:137]:
```suggestion
    distApi("com.microsoft.identity:common:3.4.5") {
```

File: adal/versioning/version.properties [hunk 2]
```diff
@@ -1,3 +1,3 @@
 #Wed Aug 01 15:24:11 PDT 2018
-versionName=3.1.2
+versionName=3.1.3-RC1
```
Comment (shahzaibj) [inline adal/versioning/version.properties:2]: This also still has the RC

File: adal/build.gradle (hunk 1 above)
Comment (AdamBJohnsonx) [reply adal/build.gradle]: No, this was a mismerge based on an out-of-date local repo.

File: adal/versioning/version.properties (hunk 2 above)
Comment (AdamBJohnsonx) [reply adal/versioning/version.properties]: Yes, though I performed the merge against origin/release/3.1.3, it still pulled from my out of date local branch.  I subsequently fixed it.

File: userappwithbroker/build.gradle [hunk 3]
```diff
@@ -80,7 +80,7 @@ dependencies {
     implementation "com.google.android.material:material:$rootProject.ext.materialVersion"
 
     localImplementation(project(':adal'))
-    distImplementation("com.microsoft.aad:adal:3.1.0") {
+    distImplementation("com.microsoft.aad:adal:3.1.3-RC1") {
```
Comment (AdamBJohnsonx) [inline userappwithbroker/build.gradle:83]:
```suggestion
    distImplementation("com.microsoft.aad:adal:3.1.3") {
```

File: adal/versioning/version.properties (hunk 2 above)
Comment (AdamBJohnsonx) [reply adal/versioning/version.properties]: And evidently not completely.

File: gradle.properties [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
 
-# When configured, Gradle will run in incubating parallel mode.
-# This option should only be used with decoupled projects. More details, visit
-# http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
-# org.gradle.parallel=true
+# This the TSL versionName...
+versionName=1.5.9
+
+# For OneAuth default abiSelection
+abiSelection=x86_64
```
Comment (shahzaibj) [inline gradle.properties:18]: nit: new line ending

**Review by shahzaibj — DISMISSED** [review]

File: gradle.properties (hunk 4 above)
Comment (AdamBJohnsonx) [inline gradle.properties:18]:
```suggestion
abiSelection=x86_64

```

Comment (AdamBJohnsonx) [reply gradle.properties]: added

**Review by shahzaibj — APPROVED** [review]

**Review by shoatman — APPROVED** [review]
Thanks Adam!

**Pull request merged**